api/docs/mac-os/couch-base/Installation.md

```diff
@@ -0,0 +1,43 @@
+# CouchBase Installation in MacOS
```

Comment:
The directory structure can be  `api/Installation/couch-base/MacOs.md` . So that the root directory will always be `Installation` followed by `OS-specific` details.

Reply:
good thought! in that case, i propose like below.

```
Installation/
    Mac/
        CouchBase.md
        XYZ.md
        ABC.md
    Windows/
        CouchBase.md
        XYZ.md
        ABC.md
```

Reply:
@ramsunvtech - agreed. Please modify accordingly.

Reply:
@jpukg @karthikeyanpa12 @naveenvignesh5 please share your thoughts

Reply:
Proposed directory structure will suffice. We will go with that.

Reply:
done